metabase/metabase
# Follow up on "Polish appearance settings"

Epic: https://github.com/metabase/metabase/issues/38699

[Discussion on Slack.](https://metaboat.slack.com/archives/C063Q3F1HPF/p1708014232222509?thread_ts=1707901949.108579&cid=C063Q3F1HPF)

### Description
These are the changes introduced in this PR:
1. The application colors and chart colors don't change after updating so it will force users to refresh the page before seeing the change taking effect. It would be more obvious than changing some colors and requiring users to still reload the page.
2. Update copy on Application colors, Custom dashboard, Custom Homepage

### How to verify

1. Go to Admin > Appearance > Color palette
3. Change any color and see that no element in the UI has its color changed yet.
4. Refresh the page
5. Elements should have new colors accordingly.

### Checklist
